src/loaders/tiff/lib/Pool.ts

```diff
@@ -48,7 +48,7 @@ export default class Pool {
       currentWorker.onmessage = event => {
         // this.workers.push(currentWorker);
         this.finishTask(currentWorker);
-        resolve(event.data[0]);
+        resolve(event.data);
```

Comment:
What happened here?

Reply:
The payload in `worker.decoder.ts` in the `postMessage` event is `result` (`Uint8Array`). I think at some point it was `[result]` but this has changed.

Reply:
How did you notice this then?  Just a little confused about how this came up.

Reply:
Remember that the decoder pool wasn't working at all, so the code here wasn't executed (silent errors). When I fixed handling of the `pool` to `readRasters`, this code was executed again and threw an error.

Reply:
It "came up" because the code was executed for the first time since making changes from `[result]` to `result` (i.e. the pool was actually in use).

Reply:
Gotcha.  Sounds good.